docs/tidb_http_api.md

```diff
@@ -0,0 +1,52 @@
+#  TiDB HTTP API
+
+`TiDBIP` is the ip of the TiDB server. `10080` is the default status port, and it can be changed in tidb.toml when started. 
+
+> 1. Get the current status of TiDB, including the connections, version and git_hash
+
+```
+curl http://{TiDBIP}:10080/status
+```
+
+> 2. Get all metrics of TiDB
+
+```
+curl http://{TiDBIP}:10080/metrics
+```
+
+> 3. Get the metadata of all regions
+
+```
+curl http://{TiDBIP}:10080/regions/meta
+```
+
+> 4. Get the information of a specific region by ID
+
+```
+curl http://{TiDBIP}:10080/regions/{regionID}
+```
+
+> 5. Get regions info from db.table
+
+```
+curl http://{TiDBIP}:10080/tables/{db}/{table}/regions
```

Comment:
There also exists a API like that to get the table disk size usage: 
`curl http://{TiDBIP}:10080/tables/{db}/{table}/disk-usage`